### PR TITLE
Update Nomad_Moogle.lua

### PR DESCRIPTION
**Pull request by some1here**
Limit Break  >>> Fatalerror66 and AlexandreCA have given their script. A HUGE <<< thank you >>> for the code.

https://forums.dspt.info/viewtopic.php?f=19&t=2703

### CODE REVIEW

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua [hunk 1]
```diff
@@ -1,57 +1,136 @@
...
+-----------------------------------
+--  Area: Ru'Lude Gardens
+--  NPC:  Nomad Moogle
 --  Type: Adventurer's Assistant
---  @pos 10.012 1.453 121.883 243
+--  @pos 10.012 1.453 121.883 243
 -----------------------------------
-package.loaded["scripts/zones/RuLude_Gardens/TextIDs"] = nil;
+package.loaded["scripts/zones/RuLude_Gardens/TextIDs"] = nil;
+package.loaded["scripts/globals/settings"] = nil;
```
Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: There is no need to nil settings.lua here.

**Comment by TeoTwawki** [comment]
As @xdemolish said in #2252 commit messages should be descriptive, more than "update thing"

**Comment by TeoTwawki** [comment]
And did you take care of the issues that prevented @AlexandreCA from pull requesting his work?
He mentioned that what you started with in #2248 was simply a copy of what he posted to forums..You should have mentioned your source if that is the case by the way.

**Comment by TeoTwawki** [comment]
Just so you know, it is not necessary to close your pull request and make a new one to make changes.

It is possible to use git amend to change your last commit, then force push to update the pull request automatically. Depending on what software you are using the method varies.

**Comment by Hozu** [comment]
Er, won't the first quest be available on a server that has the level cap at 75?

File: scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua [hunk 2]
```diff
@@ -1,57 +1,131 @@
------------------------------------
---  Area: Ru'Lude Gardens
---  NPC:  Nomad Moogle
---  Type: Adventurer's Assistant
---  @pos 10.012 1.453 121.883 243
 -----------------------------------
-package.loaded["scripts/zones/RuLude_Gardens/TextIDs"] = nil;
```
Comment (TeoTwawki) [inline scripts/zones/RuLude_Gardens/npcs/Nomad_Moogle.lua]: You originally added a nil for settings.lua which wasn't needed. now you are removing TextIDs.

You misunderstand: 
TextIDs.lua needed that line. The settings did not.
When you nil it, that means it is unloaded from memory and when you require it, it is reloaded.
TextIDs.lua is special and has to be done this way, the others are not.

**Comment by AlexandreCA** [comment]
@TeoTwawki it's not my work but I respect the work of other. Him, not really and that, I found that no correct!

**Comment by TeoTwawki** [comment]
@AlexandreCA My bad. Thought the forum post was your post.

**Comment by AlexandreCA** [comment]
@TeoTwawki correct! Fatalerror66 has given me his script. And one little thank you for the author will be apprecied . 

https://forums.dspt.info/viewtopic.php?f=19&t=2703

**Comment by TeoTwawki** [comment]
@AlexandreCA thank you, I was looking for the thread myself just now.

**Comment by TeoTwawki** [comment]
My previous comment has been edited.

**Comment by AlexandreCA** [comment]
you're welcome. No problem, but when is it updating out?

**Comment by TeoTwawki** [comment]
> Do I have to mention SE too?

Pull request comment is not the same as a commit message and doesn't go into history. also, that was rude. Citing sources is at reasonable expectation.

**Comment by TeoTwawki** [comment]
update thing
update thing
update thing
...
https://encrypted.google.com/search?hl=en&q=how%20to%20use%20git%20amend

**Comment by TeoTwawki** [comment]
> did you take care of the issues that prevented Fatalerror66 from pull requesting his work?

Still hasn't been answered.

> TextIDs.lua needed that line. The settings did not.

Still hasn't been fixed.

If a script has issues still, its not likely to be merged plain and simple.

**Comment by some1here** [comment]
Last Post I put AlexandreCA's version on. As it looked like you were requesting.

**Comment by TeoTwawki** [comment]
@some1here Was not requesting anyone's version, was asking if you had fixed the issues that prevented this from being pull requested before and to not nil the settings. Then saw the TextIDs got nil, asked to put TextIDs line back.  :-/

 I didn't see that you had swapped out the entire script.

**Comment by some1here** [comment]
That "nil" update been done before I moved to AlexandreCA's version. Just leave it as his since he had everything working. His was also simpler

**Comment by some1here** [comment]
Closing since AlexandreCA ha opened change

**Pull request closed without merging**